src/django/api/fields.py

```diff
@@ -9,6 +9,33 @@
 from rest_framework.serializers import ValidationError
 
 
+def get_polygon_geometry(geojson):
+    if type(geojson) in [str, bytes]:
```

Comment:
Not an issue, but curious why you didn't opt for isinstance (true... we're not needing to support subclasses here).

Reply:
Good point! I hadn't thought of it. Added in [ac3b493](https://github.com/azavea/iow-boundary-tool/pull/223/commits/ac3b4935671c5b69faea34c0430280e2493f461e).